src/biogeochem/CNFUNMod.F90

```diff
@@ -1608,6 +1619,44 @@ real(r8) function fun_cost_fix(fixer,a_fix,b_fix,c_fix,big_cost,crootfr,s_fix, t
   end if    ! ends up with the fixer or non-fixer decision
   
   end function fun_cost_fix
+
+
+!=========================================================================================
+  real(r8) function fun_cost_fix_Bytnerowicz_noAcc(fixer,nfix_tmin,nfix_topt,nfix_tmax,big_cost,crootfr,s_fix, tc_soisno)
+
+! Description:
+!   Calculate the cost of fixing N by nodules.
+! Code Description:
+!   This code is written to CTSM5.1 by Will Wieder 11/17/2022, modified for CLM6 11/01/2024
+
+  implicit none
+!--------------------------------------------------------------------------
+! Function result.
+!--------------------------------------------------------------------------
+! real(r8) , intent(out) :: cost_of_n   !!! cost of fixing N (kgC/kgN)
+!--------------------------------------------------------------------------
+  integer,  intent(in) :: fixer     ! flag indicating if plant is a fixer
+                                    ! 1=yes, otherwise no.
+  real(r8), intent(in) :: nfix_tmin ! As in Bytnerowicz et al. (2022)
+  real(r8), intent(in) :: nfix_topt ! As in Bytnerowicz et al. (2022)
+  real(r8), intent(in) :: nfix_tmax ! As in Bytnerowicz et al. (2022)
+  real(r8), intent(in) :: big_cost  ! an arbitrary large cost (gC/gN)
```

Comment:
There's a couple details here that I wonder about, that probably don't matter much. I'm not sure why big_cost is passed in if it's fixed? And there is a 1._r8 that doesn't seem to be needed in the question below. Also -1*s_fix should just be -s_fix. The multiply by the integer -1 doesn't add anything, and just adds complexity.

Reply:
I'm simplifying the equation by removing the two 1s that, I agree, seem redundant. Unless they represent some constant or coefficient that could later assume a different value. If it turns out that way, we can revisit later.

Reply:
I will not change how big_cost is handled for now, but I think I see that it could be set as a parameter at the module level and not need passing.